packages/media-query-list-parser/src/nodes/media-feature-comparison.ts

```diff
@@ -30,8 +30,7 @@ export function matchesComparison(componentValues: Array<ComponentValue>): false
 						return [firstTokenIndex, i];
 					}
 
-					firstTokenIndex = i;
-					continue;
+					return [i, i];
```

Comment:
For `==` it would continue here and then return on line `30` with `==` as the operator.

This however is invalid.
This function must always return when encountering `=`.

Either a lone `=` or `>=` | `<=`